framework/src/org/apache/cordova/CordovaActivity.java

```diff
@@ -147,6 +147,17 @@ public void setAuthenticationToken(AuthenticationToken authenticationToken, Stri
     }
 
     /**
+     * Specifies the HTTP auth handler for handling auth challenge requests.
+     *
+     * @param handler The HTTP auth handler.
+     */
+    public void setHttpAuthRequestHandler(HttpAuthRequestHandler handler) {
```

Comment:
If there's no direct reason to add this to CordovaActivity, I'd prefer it not added. CordovaActivity has a lot of symbols in it already, and they a little bit just add noise.